multicluster/controllers/multicluster/memberclusterannounce_controller.go

```diff
@@ -256,30 +257,40 @@ func (r *MemberClusterAnnounceReconciler) AddMember(MemberId common.ClusterID) {
 		Reason:             ReasonNeverConnected,
 	})
 
-	r.memberStatus[MemberId] = &multiclusterv1alpha1.ClusterStatus{ClusterID: string(MemberId),
+	r.memberStatus[memberID] = &multiclusterv1alpha1.ClusterStatus{ClusterID: string(memberID),
 		Conditions: conditions}
 
-	r.timerData[MemberId] = &timerData{connected: false, lastUpdateTime: time.Time{}}
+	r.timerData[memberID] = &timerData{connected: false, lastUpdateTime: time.Time{}}
+	klog.InfoS("new member is added", "member", memberID)
 }
 
-func (r *MemberClusterAnnounceReconciler) RemoveMember(MemberId common.ClusterID) {
-	defer r.mapLock.Unlock()
+func (r *MemberClusterAnnounceReconciler) RemoveMember(memberID common.ClusterID) {
 	r.mapLock.Lock()
+	defer r.mapLock.Unlock()
 
-	if _, ok := r.memberStatus[MemberId]; ok {
-		delete(r.memberStatus, MemberId)
+	if _, ok := r.memberStatus[memberID]; ok {
+		delete(r.memberStatus, memberID)
 		return
 	}
 
-	if _, ok := r.timerData[MemberId]; ok {
-		delete(r.timerData, MemberId)
+	if _, ok := r.timerData[memberID]; ok {
+		klog.InfoS("remove members", "member", memberID)
```

Comment:
"remove members" is not clear whether it's going to remove it or it has done it. Please move it to the bottom of the method and change the log to "Removed member" to indicate it has been done.
Perhaps change the log in `AddMember` to "Added member" to be consistent.

Reply:
Done